Loop/Extensions/UIAlertController.swift

```diff
@@ -125,25 +125,25 @@ extension UIAlertController {
         }
     }
 
-    /// Initializes an action sheet-styled controller for selecting a service UI.
+    /// Initializes an action sheet-styled controller for selecting a service.
     ///
     /// - Parameters:
-    ///   - serviceUITypes: An array of service UI types.
-    ///   - selectionHandler: A closure to execute when a service UI is selected.
-    ///   - service: The selected service UI type.
-    internal convenience init(serviceUITypes: [ServiceUI.Type], selectionHandler: @escaping (_ serviceUIType: ServiceUI.Type) -> Void) {
+    ///   - services: An array of services.
+    ///   - selectionHandler: A closure to execute when a service is selected.
+    ///   - identifier: The identifier of the selected service.
+    internal convenience init(services: [AvailableDevice], selectionHandler: @escaping (_ identifier: String) -> Void) {
```

Comment:
nit: Could just be me, but I find `services: [AvailableDevice]` confusing and I questioned if this was correct. Maybe a `Typealias` to define `AvailableService`?

Reply:
👍